core/string/translation_po.h

```diff
@@ -31,7 +31,7 @@
 #ifndef TRANSLATION_PO_H
 #define TRANSLATION_PO_H
 
-//#define DEBUG_TRANSLATION_PO
+#define DEBUG_TRANSLATION_PO
```

Comment:
Why?

Reply:
I accidentally left this in while converting code because Visual Studio Code's hover documentation doesn't work in code that is in a negative `#ifdef`.